app/controllers/openid_token_proxy/callback_controller.rb

```diff
@@ -0,0 +1,20 @@
+module OpenIDTokenProxy
+  class CallbackController < ApplicationController
+    def handle
+      unless code = params[:code]
+        render nothing: true, status: :bad_request
```

Comment:
I think it's probably not necessary to translate. Only developers will ever see it anyway, right? It's just to make things easier to debug.